it/topics/scripting_languages/JavaScript/README.md

```diff
@@ -0,0 +1,224 @@
+# JavaScript
+
+- [Che cos'è JavaScript?](#che-cos'è-javascript?)
```

Comment:
The link is an HTML ID - it cannot have `?` or other punctuation.

When I find the actual heading lower down then and copy the link to it, I get this, which is the correct ID:

`#che-cosè-javascript`

Reply:
```suggestion
- [Che cos'è JavaScript?](#che-cosè-javascript)
```